### PR TITLE
feat: Simplify benchmark container to only run system metrics

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/performance-tools/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Simplify the functionality of the benchmark container. The container will now only run the basic system metrics and run indefinitely. The benchmark python refactor will be in control of the duration of the run.

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
please taking the LOG_DIRECTORY env for scripts (instead of hard-coded /tmp/results) as that part could be customized in stream-dentistry scripts

**Comment by brian-intel** [comment]
> please taking the LOG_DIRECTORY env for scripts (instead of hard-coded /tmp/results) as that part could be customized in stream-dentistry scripts

/tmp/results is the internal Docker container directory. The idea was to keep this consistent and we adjust the external location with the volume mount. 
ex. -v $SOURCE_DIR/$LOG_DIRECTORY:/tmp/results

This container is for system wide metrics so it will work the same in the --pipeline and --stream-density cases. The benchmark python will handle the stream-density case.

**Comment by jim-wang-intel** [comment]
> > please taking the LOG_DIRECTORY env for scripts (instead of hard-coded /tmp/results) as that part could be customized in stream-dentistry scripts
> 
> /tmp/results is the internal Docker container directory. The idea was to keep this consistent and we adjust the external location with the volume mount. ex. -v $SOURCE_DIR/$LOG_DIRECTORY:/tmp/results
> 
> This container is for system wide metrics so it will work the same in the --pipeline and --stream-density cases. The benchmark python will handle the stream-density case.

ok; that is changing the original design of the script of removing the flexibility of config-  just you know. if that is what you want, i am fine with  that

**Review by jim-wang-intel — COMMENTED** [review]
LGTM other than changing the copyright year of both changed files

File: benchmark-scripts/Dockerfile.benchmark
Comment (jim-wang-intel) [inline benchmark-scripts/Dockerfile.benchmark:1]: change copyright year

File: benchmark-scripts/collect_platform_metrics.sh
Comment (jim-wang-intel) [inline benchmark-scripts/collect_platform_metrics.sh:1]: change copyright year

**Review by antoniomtz — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]

**Pull request merged**